modules/tenant-management/src/Volo.Abp.TenantManagement.Blazor/Pages/TenantManagement/TenantManagement.razor.cs

```diff
@@ -93,6 +100,7 @@ public class TenantInfoModel
 
         public bool UseSharedDatabase { get; set; }
 
+        [Required]
```

Comment:
This may be nullable. Does not required.

Reply:
It's only run if the checkbox is ticked so I think it's best to be Required in this case